src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/synthetic_prompt_generator.py

```diff
@@ -67,26 +67,54 @@ def _create_farewell_lines(cls) -> List[str]:
     @classmethod
     def _create_prompt_from_farewell_lines(
         cls,
-        remaining_prompt_tokens: int,
+        requested_prompt_tokens: int,
         farewell_lines: List[str],
         tokenizer: Tokenizer,
     ) -> str:
-        prompt = ""
         get_token_length = lambda text: len(tokenizer.encode(text))
 
-        sampling_lines = True
-        while sampling_lines:
-            for line in farewell_lines:
-                line_to_add = line
-                if remaining_prompt_tokens - get_token_length(line_to_add) < 0:
-                    # This will cut off a line in the middle of a word, but that's ok since an
-                    # llm should be able to handle that.
-                    line_to_add = line_to_add[: int(math.ceil(remaining_prompt_tokens))]
-                    sampling_lines = False
-                    prompt += line_to_add
-                    break
-                prompt += line_to_add
-                remaining_prompt_tokens -= get_token_length(line_to_add)
+        line_iterator = itertools.cycle(farewell_lines)
+
+        def word_generator():
+            while True:
+                next_line = next(line_iterator)
+                words = re.split("[ \n]+", next_line)
+                for word in words:
+                    yield word
+
+        word_iterator = word_generator()
+
+        # FAST ADD lines
```

Comment:
What does FAST refer to here?
Is this method just really quick?
or is that an acronym i dont know?

Reply:
just really quick. I made it lower case now

Reply:
oh sick. thanks for being on top of that.